scripts/13-ocp-iam-users.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Use this script to grant access to IBM Cloud users to ROKS cluster
+# Update the IAM_USERS list below with the list of user email addresses
+# then run the script ./13-ocp-iam-users.sh
+
+set -euo pipefail
+
+TOOLKIT_GROUP=${TOOLKIT_GROUP:-ibm-toolkit-users}
```

Comment:
You can take as input a list of groups to add the user like "TOOLKIT_GROUPS" if they don't provide a list then defaults to "ibm-toolkit-users argocd-admins self-provisioner"

 check if "self-provisioner" in the list then check if the group is present or not and created like you do and add the cluster-role to it.

Then loop over "$TOOLKIT_GROUPS" like you do.

The admin might want to add an toolkit user but has the option on not providing self-provisioning, because the adin will create the namespace for the users.